### DISCUSSION TITLE
Avoid custom "canary" version tag?

### DISCUSSION THREAD

**Discussion by mattwoodco** [nextauthjs/next-auth#1927] [Help]
Will you consider pointing these packages to their latest, exact versions?
- @next-auth/prisma-legacy-adapter
- @next-auth/typeorm-legacy-adapter

The custom "canary" version tag pointing to the these adapters [here](https://github.com/nextauthjs/next-auth/blob/main/package.json#L64) is preventing me from importing the package, due to custom version tag restrictions.

**Comment by balazsorban44** [comment]
currently they have no latest versions yet, but we don't plan changing any of their functionality so it shouldn't cause any problems. not sure what custom version tag restrictions mean
